_oasis.in

```diff
@@ -6,8 +6,8 @@ Authors:     Dave Scott
 License:     LGPL-2.1 with OCaml linking exception
 Plugins:     META (0.2)
 OCamlVersion: >= 4.01
-AlphaFeatures: ocamlbuild_more_args
-XOCamlbuildPluginTags: package(cppo_ocamlbuild)
+AlphaFeatures: ocamlbuild_more_args, compiled_setup_ml
+XOCamlbuildPluginTags: package(cppo_ocamlbuild), package(bisect_ppx.ocamlbuild)
```

Comment:
`xenopsd` doesn't seem to use that script yet, I see that xapi does though. Should I add it?